lib/puppet/version.rb

```diff
@@ -73,19 +67,19 @@ def self.version
     @puppet_version ||= PUPPETVERSION
   end
 
-  # @return [SemanticPuppet::Version] the puppet version
-  def self.parsed_version
-    @parsed_version ||= SemanticPuppet::Version.parse(version)
+  # @return [String] containing the puppet major version, e.g. "3"
+  def self.version_major
+    self.version.split('.').first
   end
 
-  # @return [String] the minor version as a  "major.minor" string
-  def self.minor_version
-    v = parsed_version
-    "#{v.major}.#{v.minor}"
+  # @return [String] containing the puppet minor version, e.g. "0"
+  def self.version_minor
```

Comment:
No, all I needed was the method that returns `MAJOR.MINOR`. @MosesMendoza suggested that it would be better if it were as close to SemVer nomenclature. I don't care either way.